error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -379,4 +380,16 @@ boolean after(Stream<T> stream) {
       return stream.allMatch(e -> test(e));
     }
   }
+
+  static final class StreamMapReduceIntSum<T> {
```

Comment:
We try to name rules after the code in the `@AfterTemplate`, which in this case would yield:
```suggestion
  static final class StreamMapToIntSum<T> {
```

Reply:
(I was wondering whether this rule should be in the `IntStreamRules` class, but since `IntStream` is only an intermediate type, I suppose not.)

Reply:
I was wondering the same, and came to the same conclusion 🙂